app/controllers/users.js

```diff
@@ -19,3 +20,20 @@ exports.signUp = async (req, res, next) => {
     return next(error);
   }
 };
+
+exports.signIn = (req, res, next) => {
+  try {
+    const { user } = res.locals;
+
+    const payload = {
```

Comment:
You must provide a date when the token will expirate.
In this case, the token could have:
```.js
{
        exp: newExpirationDate,
        id: user.id,
        iss: 'JWT'
}
```